operations/patch.go

```diff
@@ -154,7 +154,11 @@ func Patch() cli.Command {
 			if err != nil {
 				return err
 			}
-			return params.displayPatch(conf, newPatch)
+			if err = params.displayPatch(conf, newPatch); err != nil {
+				grip.Error(err)
+			}
+			params.setDefaultProject(conf)
```

Comment:
It looks like you're putting this at the end of patch just to the print statement is at the end? I don't think it'll be confusing to have the print statement come earlier, and to leave this as part of loadProject so we aren't doing duplicate work.

Reply:
I moved it to the end so that it only sets the default after we get a 200 status from the server. People sometimes pass the wrong project, so I want to only set the default if we know it's right (the diff applies cleanly on a tracked branch)